Wrappers/Python/cil/optimisation/functions/TotalVariation.py

```diff
@@ -200,8 +259,11 @@ def proximal(self, x, tau, out = None):
         tmp_q = tmp_p.copy()
         tmp_x = self.gradient.domain_geometry().allocate(0)     
         p1 = self.gradient.range_geometry().allocate(0)
-        
 
+        if self.strong_convexity_constant>0:
+            tau /= (1 + tau*self.strong_convexity_constant)
+            x /= (1 + tau*self.strong_convexity_constant)
```

Comment:
Also here we are modifying the input parameter. This is fine as long as we put back the original values at the end of the call. See a similar comment in the regulariser plugin.

Reply:
Fixed